src/building/house_service.c

```diff
@@ -118,6 +119,65 @@ void house_service_calculate_culture_aggregates(void)
                 }
             }
         }
+        
+        if (b->house_population > 0) {
+			if (b->data.house.theater) {
+				if (b->subtype.house_level >= HOUSE_SMALL_PALACE) {
+					b->data.house.theater_accessibility = 100;
```

Comment:
The reason is Grand Insula is taken as a benchmark for the model with pop 84. Starting with Small Palace population (beds in house) start to increase significantly and it would yield lover benchmark value the higher the house level (since we are dividing access level with house population). To avoid this we are assuming such spacious housing with patricians has money to get private lecturers and performers if the house has in general access to the requiring facility (theater, school, etc.)

Reply:
I see, then it makes sense.

But does that mean that houses which aren't filled with pop will reduce the rating, even if it has full access to cultural venues?

Reply:
> I see, then it makes sense.
> 
> But does that mean that houses which aren't filled with pop will reduce the rating, even if it has full access to cultural venues?

In opposite. Less the pop - higher the rating. This is in line with the base game paradigm - one building serves certain amount of people.